pkg/image/trigger/annotations/annotations_test.go

```diff
@@ -284,11 +284,8 @@ func TestAnnotationsReactor(t *testing.T) {
 	for i, test := range testCases {
 		u := &fakeUpdater{}
 		r := AnnotationReactor{Copier: kapi.Scheme, Updater: u}
-		initial, err := kapi.Scheme.DeepCopy(test.obj)
-		if err != nil {
-			t.Fatal(err)
-		}
-		err = r.ImageChanged(test.obj, fakeTagRetriever(test.tags))
+		initial := test.obj.DeepCopy()
+		err := r.ImageChanged(test.obj, fakeTagRetriever(test.tags))
```

Comment:
this is odd. The r.ImageChanged func mutates test.obj?

Reply:
i think the test guards against mutation (see the line below)